### ISSUE TITLE
Add-ons

### ISSUE THREAD

**Issue by danielmlow** [satvik-dixit/speech_emotion_recognition#1]
@satvik-dixit 

- Replace Accuracy for F1-score. Accuracy is sensitive to class imbalance. Add classification_report() to understand how the model is performing for each class and so we also get precision and recall. https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html 
- When you create a summary table where you have the different models (byols, compare, egemaps) as rows and the different datasets as columns, use the rounded F1 scores to 3 digits and populate the table automatically after each run, append the score to a list and then use the lists to build the DF that way you avoid any potential human copying errors. 
- Replace the category labels (e.g., F, A, S) with interpretable labels (e.g., fear, anger, sadness) according to each dataset. All the info to interpret results should be in the notebook.
